### DISCUSSION TITLE
Convert entire TTF font to a PNG image map.

### DISCUSSION THREAD

**Discussion by prusso630** [ImageMagick/ImageMagick#5761] [Development] [answered]
I would like to export an entire unicode font "GreekC_IV25" and export each character onto a png image map with a transparent background. Is there a way to do such a thing from terminal?

Also I tried to use the imagemagick convert command but I had some difficulty entering a unicode character code such as \x4B and have anything come out I just got an error about the arguments to -draw being invalid. Could I get an example of how to do such a thing as it relates to "GreekC_IV25"? 

Thanks...

**Comment by fmw42** [comment, marked as answer]
I think you need type your characters into a UTF-compatible text editor and save a file of each or all the characters rather than trying to use the unicode. Then provide the text file with an @ prefix and you should get your proper characters.  See https://imagemagick.org/Usage/text/#unicode.

See also

https://www.imagemagick.org/Usage/scripts/show_fonts
https://www.imagemagick.org/Usage/scripts/graphics_utf

Not sure if those will help.

**Comment by prusso630** [comment]
Those were good. I was recommended UniRed as a text editor for unicode on windows it worked so far. There are just the details for me to look up about it in docs I think.
